pychromecast/quick_play.py

```diff
@@ -5,9 +5,11 @@
 from .controllers.yleareena import YleAreenaController
 from .controllers.spotify import SpotifyController
 from .controllers.bubbleupnp import BubbleUPNPController
+from .controllers.bbciplayer import BbcIplayerController
+from .controllers.bbcsounds import BbcSoundsController
 
 
-def quick_play(cast, app_name, data):
+def quick_play(cast, app_name, data, **kwargs):
```

Comment:
Why do we need both `**kwargs` and a `data` dict?
Can Home Assistant's quickplay service pass in the needed `kwargs`?

Reply:
Not sure on your first point about whether both are needed, I will take another look. 
I have tested end-to-end (casting with and without metadata when calling the service in a script) with the changes in this PR and a one line change in the [draft PR](https://github.com/home-assistant/core/pull/51148).